sympy/vector/coordsysrect.py

```diff
@@ -98,6 +213,65 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
+        if transformation is None:
+            if parent is None:
+                if rotation_matrix is not None:
+                    rotation_equations = \
+                       lambda x, y, z: CoordSys3D._rotation_trans_equations(parent_orient, lambda x, y, z: (x, y, z))
+                else:
+                    rotation_equations = lambda x, y, z: (x, y, z)
+
+                # Information about location without parent doesn't have sense.
+                # See lines 80:99. Location is somehow treated only when parent is put.
+                # We are not able to create independent vector, so we can't assign
+                # anything to location.
+                translation_equations = lambda x, y, z: (x, y, z)
+
+                from sympy import symbols
+                x, y, z = symbols('x y z')
+                transformation = rotation_equations(*translation_equations(x, y, z))
+                lame_coefficients = _set_lame_coefficient_mapping('cartesian', (x, y, z))
+                inverse_transformation = None
+
+                if not CoordSys3D._check_orthogonality(transformation, (x, y, z)):
+                    raise ValueError("The transformation equation does not "
+                                     "create orthogonal coordinate system")
+
+            else:
+                if rotation_matrix is not None:
+                    rotation_equations = \
+                        lambda x, y, z: parent._rotation_trans_equations(parent_orient, (x, y, z))
+                else:
+                    rotation_equations = lambda x, y, z: (x, y, z)
+
+                if location is not None:
+                    translation_equations = \
+                        lambda x, y, z: parent._translation_trans_equations(parent, origin, (x, y, z))
+                else:
+                    translation_equations = lambda x, y, z: (x, y, z)
+
+                transformation = rotation_equations(*translation_equations(*parent.base_scalars()))
+                lame_coefficients = _set_lame_coefficient_mapping('cartesian', parent.base_scalars())
+                inverse_transformation = _set_inv_trans_equations('cartesian', parent.base_scalars())
+
+                if not CoordSys3D._check_orthogonality(transformation, parent.base_scalars()):
+                    raise ValueError("The transformation equation does not "
+                                     "create orthogonal coordinate system")
+        else:
+            if location is not None and rotation_matrix is not None:
+                raise ValueError
+
+            if parent is None:
+                connect_to = CoordSys3D._connect_to_standard_cartesian(transformation, variable_names)
+
+                transformation = connect_to[0]
+                lame_coefficients = connect_to[1]
+                inverse_transformation = connect_to[2]
+            else:
+                transformation = None
```

Comment:
I didn't cover this scenario. When you accept the case when `parent` is `None` I will update this part. 
I wrote about it on gitter